org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -1327,8 +1322,9 @@ public TargetPropertyType getType() {
     public enum PlatformOption implements DictionaryElement {
         NAME("name", PrimitiveType.STRING),
         BAUDRATE("baud-rate", PrimitiveType.NON_NEGATIVE_INTEGER),
-        BOARD("board", PrimitiveType.STRING);
-        
+        BOARD("board", PrimitiveType.STRING),
```

Comment:
I planned on adding these changes with the arduino lingua franca pull request with the additional platform option of port for what location to flash to.

Reply:
Sounds good!